src/Notifier.php

```diff
@@ -190,4 +198,21 @@ public function notify($exc)
 
         return $this->sendNotice($notice);
     }
+
+    /**
+     * Builds notices URL
+     *
+     * @param array $opt
+     * @return string
+     */
+    protected function buildNoticesURL(array $opt)
+    {
+        $schemeAndHost = $opt['host'];
+
+        if (!preg_match('~^https?://~i', $schemeAndHost)) {
```

Comment:
Can you please avoid using RegExp here? It is called every time notice is sent. For example, you can use

```
function substr_startswith($haystack, $needle) {
    return substr($haystack, 0, strlen($needle)) === $needle;
}
```

or build URL once and cache it in the notifier.